WPMedia/fusion-news-theme-blocks
# Tmedia 478 nav bar mobile breakpoint bug

## Description
Navigation bar renders outside of the viewport on certain mobile devices

## Jira Ticket
- [TMEDIA-478](https://arcpublishing.atlassian.net/browse/TMEDIA-478)

## Acceptance Criteria
- Items within the nav bar (background, hamburger, logo, identity buttons) are responsive and fit to the viewport size
- Background color is 100% of width
- Logo is centered and 60% (?) of width

## Test Steps

storybook 

1. Go to the chromatic page in the pr and see wide logo header nav chain block
2. You can shrink to the smallest and expand to full desktop width of the window. Notice that the image shrinks with the viewport 

live fusion 

1. Checkout this branch `git checkout TMEDIA-478-nav-bar-mobile-breakpoint-bug`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/header-nav-chain-block`
3. Go to the homepage
4. Use the emulator of the mobile viewport and use responsive mode. Shrink the homepage to the smallest widths and notice that the nav image does not stick out past what other elements do

## Effect Of Changes
### Before

at small viewport 
<img width="485" alt="Screen Shot 2021-12-13 at 16 44 01" src="https://user-images.githubusercontent.com/5950956/145901389-90234aca-7595-4a93-98b1-db35470b3e62.png">

larger viewport 

<img width="1433" alt="Screen Shot 2021-12-13 at 16 51 43" src="https://user-images.githubusercontent.com/5950956/145901749-67e38490-1f91-48a2-8f3b-3a0f45a9e0a7.png">


### After


at small viewport 
<img width="438" alt="Screen Shot 2021-12-13 at 16 44 24" src="https://user-images.githubusercontent.com/5950956/145901355-b25651b1-6166-4f6f-94c9-6359e8d8804a.png">

at larger viewport 

<img width="1429" alt="Screen Shot 2021-12-13 at 16 51 07" src="https://user-images.githubusercontent.com/5950956/145901653-492d98d0-96f0-4d51-95f5-d36014297199.png">

## Dependencies or Side Effects
- logo will shrink relative to viewport 

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
